### ISSUE TITLE
Tooltip close button is rendered as a link instead of a button

### ISSUE THREAD

**Issue by henrycatalinismith** [henrycatalinismith/zetkin#12]
## Steps to reproduce

1. Open an assignment.
2. Start VoiceOver.
3. Focus the tooltip's `I understand button`.

## Expected outcome

As this element doesn't navigate to another page, but rather changes the state of the UI on the current page, it should be announced as a button rather than a link.

## Actual outcome

The element is announced as a link.

![Screenshot of the above screen reader scenario for the benefit of sighted people unfamiliar with screen readers](https://user-images.githubusercontent.com/566159/191825111-ad433835-6d08-4373-b1bf-3d7e99d5e959.png)

## Recommendation

Use `<button>` rather than `<a>` for this one.
